server/services/zwave/lib/comClass/switchMultilevelCommandClass.js

```diff
@@ -0,0 +1,119 @@
+const { DefaultCommandClass } = require('./defaultCommandClass');
+const logger = require('../../../../utils/logger');
+const { COMMAND_CLASSES, INDEXES } = require('../constants');
+
+const SwitchMultilevelCommandClass = function SwitchMultilevelCommandClass() {
+  DefaultCommandClass.call(this);
+};
+SwitchMultilevelCommandClass.prototype = Object.create(DefaultCommandClass.prototype);
+Object.defineProperty(SwitchMultilevelCommandClass.prototype, 'constructor', {
+  value: SwitchMultilevelCommandClass,
+  enumerable: false,
+  writable: true,
+});
+
+/**
+ * @description Test if provided comClass support V4.
+ *
+ * @param {Object} comClass - The comClass zWave object.
+ *
+ * @returns {boolean} True if the current node supports V4. False otherwise.
+ *
+ * @example
+ *  const isSupportingV4 = supportV4(zWaveComClassObject);
+ */
+function supportV4(comClass) {
```

Comment:
This function name is not really explicit, which v4 is it? Gladys v4? 

From an outside point of view, it's not clear enough :)

Reply:
You are here in a method from the switchMultilevelCommandClass class. We are so talking about the V4 of the SwitchMultilevelCommandClass... I can rename it if you prefer...

Reply:
Yes, it's not clear enough